config/product/fleet.js

```diff
@@ -18,6 +18,7 @@ export function init(store) {
 
   product({
     ifHaveType:            FLEET.GIT_REPO,
+    ifHave:                IF_HAVE.FLEET,
```

Comment:
There is already a dsl key for this..

https://github.com/rancher/dashboard/blob/8f2382e48daf5caa316f378269292ae604463f9c/store/type-map.js#L1119

Reply:
Cool, I'll switch